lib/rubocop/packs.rb

```diff
@@ -63,19 +63,16 @@ def self.offenses_for(paths:, cop_names:)
     # exclusions with the closest ancestor `.rubocop_todo.yml`
     #
     sig { params(packs: T::Array[ParsePackwerk::Package]).void }
-    def self.auto_generate_rubocop_todo(packs:)
+    def self.regenerate_todo(packs:)
       offenses = offenses_for(
         paths: packs.map(&:name).reject { |name| name == ParsePackwerk::ROOT_PACKAGE_NAME },
         cop_names: config.permitted_pack_level_cops
       )
 
       offenses.group_by(&:pack).each do |pack, offenses_for_pack|
         rubocop_todo_yml = pack.directory.join(PACK_LEVEL_RUBOCOP_TODO_YML)
-        if rubocop_todo_yml.exist?
-          rubocop_todo = YAML.load_file(rubocop_todo_yml)
-        else
-          rubocop_todo = {}
-        end
+        rubocop_todo_yml.delete if rubocop_todo_yml.exist?
```

Comment:
Delete the old file to remove duplicates, files that don't exist, and files that don't have issues anymore.